### PR TITLE
[CDAP-17693] Introduce spark 3 for tests, drop spark 1, create spark …

### PR DESCRIPTION
**Pull request by tivv**
…3 runtime for cdap sandbox and standalone

### CODE REVIEW

File: kafka-plugins-0.10/pom.xml [hunk 1]
```diff
@@ -195,8 +195,8 @@
         <version>1.1.0</version>
         <configuration>
           <cdapArtifacts>
-            <parent>system:cdap-data-pipeline[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
-            <parent>system:cdap-data-streams[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
+            <parent>system:cdap-data-pipeline2_2.11[6.1.0-SNAPSHOT,7.0.0-SNAPSHOT)</parent>
```
Comment (albertshau) [inline kafka-plugins-0.10/pom.xml]: don't need to change parents

**Review by albertshau — APPROVED** [review]

File: kafka-plugins-0.10/pom.xml (hunk 1 above)
Comment (tivv) [reply kafka-plugins-0.10/pom.xml]: Yes, this is outdated, please review once again

**Pull request merged**